src/client/dfs/dfs.c

```diff
@@ -7013,10 +7015,28 @@ dfs_recreate_sb(daos_handle_t coh, dfs_attr_t *attr)
 	if (rc)
 		D_GOTO(out_prop, rc);
 
-	rc = daos_obj_close(super_oh, NULL);
+	/** relink the root object */
+	rentry.oid = roots->cr_oids[1];
+	rentry.mode = S_IFDIR | 0755;
```

Comment:
So superblock is 755 and other directories (above) are 700? What determines this?

Reply:
this is just matching what the root is originally created with. this is not inserted in lost+found like above, but is actually relinking the root object.
note that i have not paid much attention to these permissions historically in DFS since they are not really enforced beyond the DFS API and mean nothing security wise. probably need though to review what is being set for them by default in master aside from this work.